src/integrations/GA4/browser.js

```diff
@@ -83,8 +73,7 @@ export default class GA4 {
 
   init() {
     // To do :: check how custom dimension and metrics is used
-    const userId = this.analytics.userId || this.analytics.anonymousId;
-    this.loadScript(this.measurementId, userId);
+    this.loadScript(this.measurementId);
```

Comment:
@mihir-4116 can you please add comment here regarding why we are removing userId from loading script?

Reply:
added comment in loadScript and identify function